src/sgp/VObject.h

```diff
@@ -100,20 +96,10 @@ void InitializeVideoObjectManager(void);
 void ShutdownVideoObjectManager(void);
 
 // Creates and adds a video object to list
-#ifdef SGP_VIDEO_DEBUGGING
-extern UINT32 guiVObjectSize;
-
-	void PerformVideoInfoDumpIntoFile(const ST::string& filename, BOOLEAN fAppend);
-	SGPVObject* AddAndRecordVObjectFromHImage(SGPImage*, UINT32 uiLineNum, const ST::string& pSourceFile);
-	SGPVObject* AddAndRecordVObjectFromFile(const ST::string& ImageFile, UINT32 uiLineNum, const ST::string& pSourceFile);
-	#define AddVideoObjectFromHImage(a) AddAndRecordVObjectFromHImage(a, __LINE__, __FILE__)
-	#define AddVideoObjectFromFile(a)   AddAndRecordVObjectFromFile(  a, __LINE__, __FILE__)
-#else
-	SGPVObject* AddStandardVideoObjectFromHImage(SGPImage*);
-	SGPVObject* AddStandardVideoObjectFromFile(const ST::string& ImageFile);
-	#define AddVideoObjectFromHImage(a) AddStandardVideoObjectFromHImage(a)
-	#define AddVideoObjectFromFile(a)   AddStandardVideoObjectFromFile(a)
-#endif
+SGPVObject* AddStandardVideoObjectFromHImage(SGPImage*);
+SGPVObject* AddStandardVideoObjectFromFile(const ST::string& ImageFile);
+#define AddVideoObjectFromHImage(a) AddStandardVideoObjectFromHImage(a)
+#define AddVideoObjectFromFile(a)   AddStandardVideoObjectFromFile(a)
```

Comment:
These defines are no longer needed, just remove the Standard from the function names.